src/ui/display.c

```diff
@@ -231,7 +231,7 @@ bool check_output_index_state() {
 }
 
 void inplace_selection_sort(size_t len, uint8_t* list) {
-    size_t i, j, position;
+    size_t i = 0, j = 0, position = 0;
```

Comment:
We were receiving `core.UndefinedBinaryOperatorResult` in the `if` below due to using uninitialized variables.
This is a false-positive since we initialize the variables before using them.

We can silence the error by either:

- initializing the variables here.
- remove this line and initialize the variable in the `for` statement `for(size_t i = 0...)`